wait/host.go

```diff
@@ -613,16 +621,23 @@ func (a *HostAwaitility) WaitForNotification(name string, criteria ...Notificati
 // WaitUntilNotificationDeleted waits until the Notification with the given name is deleted (ie, not found)
 func (a *HostAwaitility) WaitUntilNotificationDeleted(name string) error {
 	return wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
-		notification := &toolchainv1alpha1.Notification{}
-		if err := a.Client.Get(context.TODO(), types.NamespacedName{Namespace: a.Namespace, Name: name}, notification); err != nil {
+		obj := &toolchainv1alpha1.NotificationList{}
+		if err := a.Client.List(context.TODO(), obj); err != nil {
```

Comment:
same question here about adding a label on the notifications.

Reply:
Updated as part of this commit: fa327b0e280198b8ac1cc89c5a1053a4763aba95